virtualmachine/gcp/vm.go

```diff
@@ -56,17 +66,127 @@ type VM struct {
 
 	AccountFile  string
 	account      accountFile
-	SSHCreds     ssh.Credentials // privateKey is required for GCE
+	SSHCreds     ssh.Credentials
```

Comment:
Why removing these one-liner description? Is it not valid anymore?

Reply:
Yes, it is not valid. We don't need private key to launch an instance. What we need is the public key and private key should always remain with the user.